messages/jsonschema/scripts/templates/ruby.rb.erb

```diff
@@ -34,6 +37,15 @@ module Cucumber
         @<%= underscore(property_name) %> = <%= underscore(property_name) %>
       <%- end -%>
       end
+      <%- schema['properties'].each do |property_name, property| -%>
+        <%- if property['items'] && property['items']['$ref']  -%>
```

Comment:
This can be heavily optimised to use `#dig`

`property.dig('items', $ref)` will return either `nil` or the value from the digging in. It's also slightly more optimal than doing double fetch iirc